announcements/20211002.md

```diff
@@ -0,0 +1,28 @@
+10 October 2021 Call For Raku Infrastructure Volunteers
```

Comment:
Wrong date?

Reply:
Sure it is! It was supposed to be month, initially...